sentences/de/light_HassTurnOff.yaml

```
@@ -5,5 +5,6 @@ intents:
       - sentences:
           - "Schalte (das Licht | (die | alle) Lichter) <area> <turn_off>"
           - "Schalte <area> (das Licht | (die | alle) Lichter) <turn_off>"
+          - "(mach | mache) <area> (das Licht | (die | alle) Lichter) <turn_off>"
```

Similar to the existing sentences above:
```suggestion
          - "(mach | mache) <area> (das Licht | (die | alle) Lichter) <turn_off>"
          - "(mach | mache) (das Licht | (die | alle) Lichter) <area> <turn_off>"
```